### DISCUSSION TITLE
Is it possible to make a commit for a specific past date?<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by Rustam200391** [community/community#101703] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Is it possible to make a commit for a specific past date?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Rustam200391 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by Rustam200391** [comment]
I executed the command and nothing happened. Next, I performed the following git commit --amend --date="Jan 28 14:00:00 2024 +0300" and git push --force. But the changes for the specified date were also not displayed on the github
